runtime/compiler/trj9/env/VMJ9.cpp

```diff
@@ -8695,6 +8745,20 @@ TR_J9SharedCacheVM::getResolvedMethods(TR_Memory *trMemory, TR_OpaqueClassBlock
       TR_J9VM::getResolvedMethods(trMemory, classPointer, resolvedMethodsInClass);
    }
 
+
+TR_ResolvedMethod *
+TR_J9SharedCacheVM::getResolvedMethodForNameAndSignature(TR_Memory * trMemory, TR_OpaqueClassBlock * classPointer,
+                                                         const char* methodName, const char *signature)
+   {
+   TR::Compilation* comp = _compInfo->getCompInfoForCompOnAppThread() ? _compInfo->getCompInfoForCompOnAppThread()->getCompilation() : _compInfoPT->getCompilation();
+   TR_ASSERT(comp, "Should be called only within a compilation");
+   if (((TR_ResolvedRelocatableJ9Method *)comp->getCurrentMethod())->validateArbitraryClass(comp, (J9Class *)classPointer))
+      return TR_J9VM::getResolvedMethodForNameAndSignature(trMemory, classPointer, methodName, signature);
+   else
+      return 0;
```

Comment:
nullptr is the preferred way of expressing null pointers in c++11.  Is there any platform that cannot handle it?

Reply:
> nullptr is the preferred way of expressing null pointers in c++11. Is there any platform that cannot handle it?

No, not all our platforms support it. However once they do it will make searching for such places to change much easier.